### PR TITLE
update env vars to latest OW code

### PR DESCRIPTION
**Pull request by tysonnorris**
updating env vars to match latest OW code
skip cli download when PROJECT_HOME is defined
use init.sh for cmd to start containers
use latest db schema
use rm -f for removing containers

### CODE REVIEW

**Comment by tysonnorris** [comment]
@mcdan not sure how to add you as a reviewer, but... could you take a look at this and see if it works for you? You may need to pull latest OW images

**Comment by mcdan** [comment]
@tysonnorris This built cleanly for me thanks.

File: docker-compose/Makefile [hunk 1]
```diff
@@ -67,17 +67,21 @@ docker:
...
-		cd $(PROJECT_HOME)/bin ; \
-    unzip -o wsk.zip; \
-	elif [ "$(UNAME_STR)" = "Linux" ]; then \
-	  echo "downloading cli for linux" ; \
-	  curl -o $(PROJECT_HOME)/bin/wsk.tgz -L https://github.com/apache/incubator-openwhisk-cli/releases/download/latest/OpenWhisk_CLI-latest-linux-amd64.tgz ; \
-		cd $(PROJECT_HOME)/bin ; \
-		tar -xf wsk.tgz ; \
-	fi; 
+	if [ "$(PROJECT_HOME)" = "./openwhisk-master" ]; then \
+        if [ "$(UNAME_STR)" = "Darwin" ]; then \
```
Comment (ddragosd) [inline docker-compose/Makefile]: :+1:

**Comment by ddragosd** [comment]
@mcdan thanks for the extra check !

**Pull request merged**